README.md

```diff
@@ -25,18 +25,20 @@ All the links are monitored and tested with [awesome_bot](https://github.com/dkh
 - [Where to start ?](#where-to-start-)
 - [Tools](#tools)
 	- [Continuous Integration / Continuous Delivery](#continuous-integration--continuous-delivery)
+		- [CI Services](#ci-services)
 	- [Deployment and Infrastructure](#deployment-and-infrastructure)
 	- [Developer Tools](#developer-tools)
 	- [Development Environments](#development-environments)
 	- [Docker Compose file](#docker-compose-file)
 	- [Dockerfile](#dockerfile)
 	- [Garbage Collection](#garbage-collection)
 	- [Hosting Images (registries)](#hosting-images-registries)
-	- [Image Builder](#image-builder)
 	- [Images](#images)
+	- [Image Builder](#image-builder)
 	- [Linter / Validator](#linter--validator)
 	- [Local Container Manager](#local-container-manager)
 	- [Monitoring & Logging](#monitoring--logging)
+		- [Monitoring & Logging Services](#monitoring-&-logging-services)
```

Comment:
@hsorellana I found more of these, you should delete all the '&' symbols in the anchor tags to make linking work. So,
```markdown
[Monitoring & Logging Services](#monitoring-&-logging-services)
```
becomes
```markdown
[Monitoring & Logging Services](#monitoring--logging-services)
```